src/components/OrderFilters.vue

```diff
@@ -0,0 +1,158 @@
+      <ion-chip id="open-order-promise-date-modal" slot="end">
+        <ion-input v-model="query.promisedDate" @ionChange="updateAppliedFilters(query.promisedDate, 'promisedDate')" type="date" />
+        <ion-icon :icon="close" v-if="query.promisedDate" @click="updateAppliedFilters('', 'promisedDate')"/>
+      </ion-chip>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Auto cancel date") }}</ion-label>
+      <ion-chip id="open-order-auto-cancel-date-modal" slot="end">
+        <ion-input v-model="query.autoCancelDate" @ionChange="updateAppliedFilters(query.autoCancelDate, 'autoCancelDate')" type="date" />
+        <ion-icon :icon="close" v-if="query.autoCancelDate" @click="updateAppliedFilters('', 'autoCancelDate')"/>
+      </ion-chip>
+    </ion-item>
+  </ion-list>
+  <ion-list>
+    <ion-list-header>{{ $t("Type") }}</ion-list-header>
+    <ion-item>
+      <ion-label>{{ $t("Store pickup") }}</ion-label>
+      <ion-checkbox :checked="query.storePickup" @ionChange="updateAppliedFilters($event['detail'].checked, 'storePickup')"/>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Ship from store") }}</ion-label>
+      <ion-checkbox :checked="query.shipFromStore" @ionChange="updateAppliedFilters($event['detail'].checked, 'shipFromStore')"/>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Pre-order") }}</ion-label>
+      <ion-checkbox :checked="query.preOrder" @ionChange="updateAppliedFilters($event['detail'].checked, 'preOrder')"/>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Back order") }}</ion-label>
+      <ion-checkbox :checked="query.backOrder" @ionChange="updateAppliedFilters($event['detail'].checked, 'backOrder')"/>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Unfillable") }}</ion-label>
+      <ion-checkbox :checked="query.unfillable" @ionChange="updateAppliedFilters($event['detail'].checked, 'unfillable')"/>
+    </ion-item>
+  </ion-list>
+  <ion-list>
+    <ion-list-header>{{ $t("Fulfillment") }}</ion-list-header>
+    <ion-item>
+      <ion-label>{{ $t("Status") }}</ion-label>
+      <ion-select :value="query.status" @ionChange="updateAppliedFilters($event['detail'].value, 'status')" interface="popover">
+        <ion-select-option v-for="status in orderStatusOptions" :key="status" :value="status">{{ status }}</ion-select-option>
+      </ion-select>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Shipping method") }}</ion-label>
+      <ion-select :value="query.shippingMethod" @ionChange="updateAppliedFilters($event['detail'].value, 'shippingMethod')" interface="popover">
+        <ion-select-option v-for="method in shippingMethodOptions" :key="method" :value="method">{{ method }}</ion-select-option>
+      </ion-select>
+    </ion-item>
+    <ion-item>
+      <ion-label>{{ $t("Ship from location") }}</ion-label>
+      <ion-select :value="query.shipFromLocation" @ionChange="updateAppliedFilters($event['detail'].value, 'shipFromLocation')" interface="popover">
```

Comment:
I think here we should use facilityTypeId values

Reply:
Sir, used facilityTypeId as a hardcoded value.